src/registry_schemas/schemas/court_order.json

```diff
@@ -37,6 +37,10 @@
                     "minLength": 0,
                     "maxLength": 2000,
                     "description": "A brief note to explain the purpose of the Court Order."
+                },
+                "fileKey": {
+                    "type": "string",
+                    "title": "The Identifier for court order file in file server"
```

Comment:
Should this, or any of the other properties, be "required" ?

Reply:
Custom validation is in legal-api

Reply:
Yes but required fields can be validated by this schema. Are any fields required for a court order?

Reply:
Required fields are different according to the type of filing. For Court order filing (staff), orderDetails is required and its not required in alteration or dissolution.

Reply:
Oh, gotcha.

OK, lgtm.